policy/Object.yaml

```diff
@@ -0,0 +1,392 @@
+funcs:
+    objectId: # the index of deepCoadd_obj IS the objectId
+        functor: Index
+    parentObjectId:
+        functor: Column
+        args: parent
+        dataset: ref
+    coord_ra:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        functor: RAColumn
+        dataset: ref
+    coord_dec:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        # coord_dec because "dec" is reserved in most SQL DBs
+        functor: DecColumn
+        dataset: ref
+    Ra:
+        functor: RAColumn
+        dataset: meas
```

Comment:
Ditto re “A”

Reply:
I don't a problem. `RAColumn` is the name of Tim's Functo,r and the column names turn into `gRa`, `rRa`, `iRa`as specified by cat and the dpdd.